src/zui/ZUIEditor/extensions/ButtonExtension.tsx

```diff
@@ -108,6 +99,34 @@ class ButtonExtension extends NodeExtension<ButtonOptions> {
       return true;
     };
   }
+
+  /* eslint-disable @typescript-eslint/ban-ts-comment */
+  //@ts-ignore
+  @command()
+  toggleButton(): CommandFunction {
+    return (props) => {
+      const { dispatch, state, tr } = props;
+      const newNode = this.type.create(
+        null,
+        this.type.schema.text('Foo button')
```

Comment:
I'm assuming this is either left in here accidentally, or intentionally left to be internationalised at a later point?

Reply:
Yea, this is definitely an embarrassing mistake. The string should be passed in either as an option to the extension, or as an argument to the command, both of which should allow internationalization quite easily since both are invoked from a react component that have access to `useMessages()`.